app/controllers/admin/AdminController.php

```diff
@@ -155,9 +155,14 @@ public function applicationsShow($id)
     // would significantly reduce the number of SQL queries needed, as well
     // as the number of variables passed to the view.
     $application = Application::getUserApplication($id);
-    $profile = Profile::getUserProfile($id);
+    //only load the user profile if it has been filled out
+    if (Profile::where('user_id', '=', $id)->exists()) {
+      $profile = Profile::getUserProfile($id);
+    }
```

Comment:
but is that what was causing the error? doesn't an if around the profile statment in the blade fix it?
if not, getUserProfile should have a failsafe, since that's doing basically the same query as the statement above it